Resources/Computing/SingularityComputingElement.py

```diff
@@ -54,6 +54,9 @@
 echo "Finishing inner container wrapper scripts at `date`."
 
 """
+# Path to a directory on CVMFS to use as a fallback if no
+# other version found: Only used if node has user namespaces
+FALLBACK_SINGULARITY = "/cvmfs/oasis.opensciencegrid.org/mis/singularity/current/bin"
```

Comment:
Do we need this now a user namespaces build of `singularity` is in DIRACOS?

Reply:
Ah, I hadn't realised it was in DIRACOS, although I'm of the opinion that it doesn't hurt to have another fallback.

Reply:
Would it make sense to make this configurable?

Reply:
It's already configurable on a per CE basis:
```python
  def __hasSingularity(self):
    """ Search the current PATH for an exectuable named singularity.
        Returns True if it is found, False otherwise.
    """
    if self.ceParameters.get('ContainerBin'):
      binPath = self.ceParameters['ContainerBin']
      if os.path.isfile(binPath) and os.access(binPath, os.X_OK):
        self.__singularityBin = binPath
        self.log.debug('Use singularity from "%s"' % self.__singularityBin)
```

Reply:
Please document: https://dirac.readthedocs.io/en/latest/AdministratorGuide/Configuration/ConfReference/Resources/Computing/index.html#singularity-ce-parameters :grin: